### ISSUE TITLE
Add systemd startup script

### ISSUE THREAD

**Issue by berghaus** [hep-gc/cloud-scheduler#463]
CentOS 7 moved away from upstart to systemd. As I understand that is a general trend most Linux flavours are following. We should provide a systemd script to control the cloud scheduler service.

**Comment by berghaus** [comment]
The startup script works under systemd as well. Do you think it is useful to keep in mind that we may need to write the systemd scripts eventually?

**Comment by mhpx** [comment]
yeah, we should probably make a proper systemd one at some point. If the existing script works that makes it a low priority though.
